ci/snapcraft.yaml

```diff
@@ -1,21 +0,0 @@
-name: electron-app
```

Comment:
🤔 I'm actually surprised that the tests for Snapcraft are passing without the .yml file anymore. Maybe the default .yml we use in the installer packager is enough now. Could we look at the test output a bit and just make sure that we're not seeing false positives?

Reply:
These tests pass because we never used the old snapcraft.yaml either :P - it was referenced in the "linux-specific setup steps" dead code referenced above, and used to create a new directory, pull in snap parts into it, then delete that directory in order to cache those parts somewhere else